addons/csw/functions/fnc_unloadAmmo.sqf

```diff
@@ -0,0 +1,93 @@
+/*
+ * Author: TCVM
+ * Unloads ammo out of the CSW and places it onto the ground
+ *
+ * Arguments:
+ * 0: CSW <OBJECT>
+ * 1: Slow Unload <BOOL>
+ * 2: Unload All Ammo <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [CSW, false, false] call ace_csw_fnc_unloadAmmo
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params["_csw", "_slowUnload", "_allAmmo"];
+
+private _weaponPos = _csw getRelPos[1.5, 270];
```

Comment:
These seem very specific values. Could you move `[1.5, 270]` to a macro so we can label it, just like a variable?